test/specs/registerTest.smoke.js

```diff
@@ -0,0 +1,14 @@
+import { expect } from '@wdio/globals';
+import MyAccountPage from '../pageobjects/myaccount.page.js';
+
+//register user test
+describe('Register User', () => {
+
+    it('should register a new user', async () => {
+        await MyAccountPage.open();
+        await MyAccountPage.register(process.env.REG_USERNAME, process.env.REG_EMAIL, process.env.REG_PASSWORD);
+
+        const alertMessage = await MyAccountPage.getAlertMessageText();
```

Comment:
Should this method name be changed?
Here is a quick tip, when renaming variables/methods in VS Code use the `Rename Symbol` option (fn + F2). This will make sure to change the name in all occurrences.